pkgs/development/tools/frink/default.nix

```diff
@@ -0,0 +1,35 @@
+{ pkgs }:
+with pkgs;
+let
+  jar = fetchurl {
+    url = "https://frinklang.org/frinkjar/frink.jar";
+    sha256 = "sha256-7pYxQnnOjOSBnk0n/Jt78sNisDKTmDqxqmsPakil5ig=";
+  };
+in stdenv.mkDerivation rec {
+  name = "frink";
+  src = ./.;
```

Comment:
Do this like ditaa

https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/graphics/ditaa/default.nix